### DISCUSSION TITLE
[버그] 지갑연결이 안됩니다.

### DISCUSSION THREAD

**Discussion by JKLEE0008** [Chia-Network/chia-blockchain#5338] [Support]
**Describe the bug**
A clear and concise description of what the bug is. (If what you are experiencing is NOT a bug but instead a support issue, please open a Discussion instead!)

**To Reproduce**

1. 윈도우 바탕화면에 chia 실행
2. 사진처럼 계속해서 지갑찾는중에서 넘어가지를 않음

![aa](https://user-images.githubusercontent.com/84335113/118510763-38123380-b76c-11eb-800f-3dbbf4616abf.png)

- OS: windows 10 pro
- CPU: Intel 8700k + 32gb

**Additional context**

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
